src/content/docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx

```diff
@@ -32,674 +34,258 @@ To install the Kubernetes integration using Helm, you will need your New Relic a
+    integration.
+
+    Additionally, New Relic provides
+    [`nri-bundle`](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle), a chart which pulls a
+    selectable set of the charts mentioned above. `nri-bundle` can also install Kube State Metrics for you if needed.
+
+    We recommend using the [`nri-bundle`](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle) chart
+    for Kubernetes deployments, as it provides the easiest way to select, install, and upgrade individual components at your will.
+
+    ### Installing and configuring `nri-bundle` with Helm
 
     1. Set the cluster where you want to install the agent:
 
-       ```
-       kubectl config set-cluster <var>DESIRED_CLUSTER</var>
-       ```
+    ```
+    kubectl config set-cluster <var>DESIRED_CLUSTER</var>
+    ```
+
+    To see the available clusters, run `kubectl config get-clusters`.
 
-       To see the available clusters, run `kubectl config get-clusters`.
     2. Add the New Relic Helm charts repo:
 
-       ```
-       helm repo add newrelic https://helm-charts.newrelic.com
-       ```
-    3. Make sure everything is configured properly in the chart by running the following command. (This step uses the `--dry-run` and `--debug` switches and therefore the agent is not installed.)
-
-       ```
-       helm install newrelic/newrelic-infrastructure \
-           --dry-run \
-           --debug \
-           --set licenseKey=<var>YOUR_NEW_RELIC_LICENSE_KEY</var> \
-           --set cluster=<var>K8S_CLUSTER_NAME</var> \
-           --set config.custom_attributes.cluster=<var>K8S_CLUSTER_NAME</var> \
-           --generate-name
-       ```
+    ```
+    helm repo add newrelic https://helm-charts.newrelic.com
+    ```
+
+    3. Make sure everything is configured properly in the chart by running the following command. This step uses the
+    `--dry-run` and `--debug` switches and therefore the agent isn't installed.
+
+    ```
+    helm upgrade --install newrelic newrelic/nri-bundle \
+    --dry-run \
+    --debug \
+    --set global.licenseKey=<var>YOUR_NEW_RELIC_LICENSE_KEY</var> \
+    --set global.cluster=<var>K8S_CLUSTER_NAME</var> \
+    --set infrastructure.enabled=true \
+    --set prometheus.enabled=true \
+    --set webhook.enabled=true \
+    --set ksm.enabled=<var>false</var> \
+    --set kubeEvents.enabled=true \
+    --set logging.enabled=true
+    ```
+
+    You can flip the `ksm.enabled` flag to `true` if KSM isn't already present in your cluster.
+
     4. Install the Kubernetes integration:
 
-       ```
-       helm install newrelic/newrelic-infrastructure \
-           --set licenseKey=<var>yYOUR_NEW_RELIC_LICENSE_KEY</var> \ 
-           --set cluster=<var>K8S_CLUSTER_NAME</var> \ 
-           --set config.custom_attributes.cluster=<var>K8S_CLUSTER_NAME</var> \
-           --generate-name
-       ```
+    ```
+    helm upgrade --install newrelic newrelic/nri-bundle \
```

Comment:
shouldn't this be `install`?

Reply:
I set is as `upgrade --install` just to avoid repeating the same command at the end of the page (in the upgrade instructions). But we can split it into install and upgrade if you think it looks nicer.

Reply:
ups I missed the --install flag